pages/windows/cmstp.md

```diff
@@ -3,34 +3,26 @@
 > A command-line tool for managing connection service profiles.
 > More information: <https://docs.microsoft.com/windows-server/administration/windows-commands/cmstp>.
 
-- Install a specific profile:
+- Install the specified profile:
 
 `cmstp "{{path/to/profile}}"`
 
-- Install without creating a desktop shortcut:
+- [U]ninstall the specified profile:
```

Comment:
```suggestion
- [u]ninstall the specified profile:
```

Reply:
Resolved as: `Uninstall the specified profile`.